// react-native-calendars/src/agenda/reservation-list/style.js
import {StyleSheet, Dimensions} from 'react-native';
import * as defaultStyle from '../../style';
import {Device, DeviceModel} from '../../../../../src/helpers'

const STYLESHEET_ID = 'stylesheet.agenda.list';

export default function styleConstructor(theme = {}) {
  const appStyle = {...defaultStyle, ...theme};
  // i don't which is for which
  var left = Device.leftAlignmentForDevice
  return  StyleSheet.create({
    container: {
      flexDirection: 'column'
    },
    dayNum: {
      fontSize: 28,
      fontWeight: '200',
      color: appStyle.agendaDayNumColor
    },
    dayText: {
      fontSize: 22,
      fontWeight: '600',
      color: appStyle.agendaDayTextColor,
      marginTop: -5,
      textAlign: 'right',
      left: left, // toheeb - date alignment
      backgroundColor: 'rgba(0,0,0,0)',
      width: '100%',
      alignItems: 'center'
    },
    day: {
      width: 500,
      justifyContent: 'center',
      marginTop: 32
    },
    today: {
      color: appStyle.agendaTodayColor
    },
    ...(theme[STYLESHEET_ID] || {})
  });
}

// toheeb - change the day e.g. Saturday, February 3
