// react-native-calendars/src/style.js
import {Platform} from 'react-native';

export const foregroundColor = '#d8d1c0';
export const backgroundColor = '#ffffff'; // #f4f4f4
export const separatorColor = '#e8e9ec';

export const processedColor = '#a7e0a3';
export const processingColor = '#ffce5c';
export const failedColor = 'rgba(246, 126, 126,1)';

export const textDefaultColor = '#2d4150';
export const textColor = '#43515c';
export const textLinkColor = '#00adf5';
export const textSecondaryColor = '#7a92a5';

export const textDayFontFamily = 'System';
export const textMonthFontFamily = 'System';
export const textDayHeaderFontFamily = 'System';

export const textDayFontSize = 16;
export const textMonthFontSize = 16;
export const textDayHeaderFontSize = 13;

export const calendarBackground = foregroundColor;
export const textSectionTitleColor = '#48849d';
export const selectedDayBackgroundColor = textLinkColor;
export const selectedDayTextColor = foregroundColor;
export const todayTextColor = '#629db7';
export const dayTextColor = textDefaultColor;
export const textDisabledColor = '#f2f2f2';
export const dotColor = textLinkColor;
export const selectedDotColor = foregroundColor;
export const arrowColor = textLinkColor;
export const monthTextColor = textDefaultColor; // e.g May 2018 in the calendar view
export const agendaDayTextColor = '#7a92a5';
export const agendaDayNumColor = '#48849d';
export const agendaTodayColor = '#629db7';
export const agendaKnobColor = Platform.OS === 'ios' ? '#413e41' : '#413e41';

// toheeb bunch of colors
